portal/views/research_study.py

```diff
@@ -0,0 +1,89 @@
+from flask import Blueprint, jsonify
+
+from ..extensions import oauth
+from ..models.organization import (
+    OrgTree,
+    OrganizationResearchProtocol,
+)
+from ..models.research_protocol import ResearchProtocol
+from ..models.research_study import ResearchStudy
+from ..models.role import ROLE
+from ..models.user import get_user
+from .crossdomain import crossdomain
+
+
+research_study_api = Blueprint('research_study_api', __name__)
+
+
+@research_study_api.route('/api/user/<int:user_id>/research_study')
+@crossdomain()
+@oauth.require_oauth()
+def rs_for_user(user_id):
+    """Returns simple JSON for research study user is associated with
+
+    ---
+    tags:
+      - User
+      - ResearchStudy
+    operationId: rs_for_user
```

Comment:
I think this appears in in title for an API in SwaggerUI and may not be the most obvious
```suggestion
    operationId: research_studies_for_user
```

Reply:
strange - this didn't show in the github ui till after i hit merge.  tempting to change in develop being a comment and all :)